### PR TITLE
Update informations for 6th meetup

### PR DESCRIPTION
**Pull request by alex3165**
Quick typos check?

### CODE REVIEW

File: assets/meta.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
   "title": "The Reactivate London Meetup",
-  "description": "Join us for our fifth meetup with talks ranging from React to this instalment's special topic: Games! Pizza and drinks on the house, of course.",
-  "date": "Aug 15th",
+  "description": "Join us for our sixth meetup with an Elm talk and our first Reactivate quiz! Pizza and drinks on the house, of course.",
```
Comment (elischutze) [inline assets/meta.json]: - can we change "pizza and drinks" to "Food and Drinks" 
- Elm talk to Elm talk/crash course

File: assets/speakers.json [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-    "avatar": "/static/Johannes.jpg",
-    "link": "https://twitter.com/frostney_",
-    "name": "Johannes Stein",
-    "title": "Let's develop a React-powered game!",
-    "text": "This talk is about React-powered game development where Johannes will be live coding a PIRATE game throughout his talk. With each example he will be implementing a new feature towards a playable prototype.",
+    "avatar": "/static/Mario.jpg",
+    "link": "https://twitter.com/realmario",
+    "name": "Mario",
+    "title": "Building unbreakable webapps fast: crash course with Elm",
+    "text": "Elm is a delightful functional language for reliable webapps. It compiles to JavaScript, has great performance, no runtime exceptions, and can be embeded easily into existing JS projects without the need for huge rewrites or time investment. We'll be taking a practically focused crash course tour of Elm, ideally you'll walk away being able to put Elm into production either from scratch or in an existing project. We'll do our best to have some hands-on hacking together. The audience will also engage interactively with live feedback/reactions powered by the presentation, which is an Elm app itself!",
```
Comment (elischutze) [inline assets/speakers.json]: web apps * (two words)

File: assets/speakers.json [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
   },
   {
-    "avatar": "/static/Johannes.jpg",
-    "link": "https://twitter.com/frostney_",
-    "name": "Johannes Stein",
-    "title": "Let's develop a React-powered game!",
-    "text": "This talk is about React-powered game development where Johannes will be live coding a PIRATE game throughout his talk. With each example he will be implementing a new feature towards a playable prototype.",
+    "avatar": "/static/Mario.jpg",
+    "link": "https://twitter.com/realmario",
+    "name": "Mario",
```
Comment (elischutze) [inline assets/speakers.json]: Mario Rogic * (according to Linked IN)

**Review by elischutze — CHANGES_REQUESTED** [review]

File: assets/speakers.json (hunk 3 above)
Comment (kitten) [reply assets/speakers.json]: We should probably just ask quickly to be sure :) it’s a name afterall

File: assets/meta.json (hunk 1 above)
Comment (kitten) [reply assets/meta.json]: Is it something different than pizza this time? I’d love that haha

Also, talk/crash course is a little clunky. Can we settle on one or the other?

Comment (elischutze) [reply assets/meta.json]: No, it's probably pizza but I'd like the option

**Pull request merged**